contracts/contracts/OracleRequestRecipient.sol

```diff
@@ -195,7 +195,7 @@ contract OracleRequestRecipient is
     receive() external payable {}
 
     /**
-     * @notice Withdraw ETH to reover stuck funds
+     * @notice Withdraw ETH to recover stuck funds
      */
     function retrieveLostTokens(address receiver) external onlyOwner {
```

Comment:
Compared to the other lost token recovery functions, is there a specific reason for not having a parameter "amount" here?